charts/ingress-nginx/values.yaml

```diff
@@ -555,7 +555,8 @@ controller:
       ##
       priorityClassName: ""
       podAnnotations: {}
-      nodeSelector: {}
+      nodeSelector:
+        kubernetes.io/os: linux
```

Comment:
I don't think this should be set as the default value, and this may affect users who do not have this label

Reply:
If you only have linux nodes this setting has no effect. Also this has already been done for one field in PR #6305 and did not affect existing users. Could you please elaborate on in which scenario it would affect existing users?

Reply:
I guess it makes sense, as the other componentes (ingress, default-backend) also have the nodeSelector.

Let's  release the CI, and watch :)